tools/monitoring/README.md

```diff
@@ -0,0 +1,22 @@
+# Knative Monitoring
```

Comment:
doc header

Reply:
Is there a way to add doc headers to the README.md so it doesn't show up when the md file is rendered? 

I couldn't find a README.md file that has the doc header. 
https://raw.githubusercontent.com/knative/test-infra/master/ci/prow/README.md

What would be the right way to add this? License info in plain text before the first heading?

Reply:
oops. did not realize that. ignore my comment..